utils/src/ooinstall/cli_installer.py

```diff
@@ -10,7 +10,7 @@
 from pkg_resources import parse_version
 from ooinstall import openshift_ansible
 from ooinstall.oo_config import OOConfig
-from ooinstall.oo_config import OOConfigInvalidHostError
+from ooinstall.models import InvalidHostError
 from ooinstall.oo_config import Host, Role
```

Comment:
Very strange, I'll have to assume some kind of chained imports are in place here, but I'll fix it regardless.